Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -974,44 +974,46 @@ def com_google_fonts_check_061(ttFont):
   id = 'com.google.fonts/check/062'
 )
 def com_google_fonts_check_062(ttFont):
-  """Is GASP table correctly set?"""
+  """Is 'gasp' table set to optimize rendering?
+  
+  Traditionally version 0 GASP tables were set
+  so that font sizes below 8 ppem had no grid
+  fitting but did have antialiasing. From 9-16
+  ppem, just grid fitting. And fonts above
+  17ppem had both antialiasing and grid fitting
+  toggled on. The use of accelerated graphics
+  cards and higher resolution screens make this
+  approach obsolete. Microsoft's DirectWrite
+  pushed this even further with much improved
+  rendering built into the OS and apps. In this
+  scenario it makes sense to simply toggle all
+  4 flags ON for all font sizes.
+  """
   try:
     if not isinstance(ttFont["gasp"].gaspRange, dict):
-      yield FAIL, "GASP.gaspRange method value have wrong type."
+      yield FAIL, "'gasp' table has no values."
     else:
       failed = False
       if 0xFFFF not in ttFont["gasp"].gaspRange:
-        yield FAIL, "GASP does not have 0xFFFF gaspRange."
+        yield WARN, "'gasp' table does not have a value for all"
+                    " sizes (gaspRange 0xFFFF). It should be set to 1."
       else:
         for key in ttFont["gasp"].gaspRange.keys():
           if key != 0xFFFF:
-            yield FAIL, ("GASP should only have 0xFFFF gaspRange,"
-                          " but {} gaspRange was also found."
-                          "").format(hex(key))
+            yield WARN, ("'gasp' table has a gaspRange of {} that"
+                         " may be unneccessary.").format(hex(key))
             failed = True
           else:
             value = ttFont["gasp"].gaspRange[key]
             if value != 0x0F:
               failed = True
-              yield WARN, (" All flags in GASP range 0xFFFF (i.e. all font"
-                           " sizes) must be set to 1.\n"
-                           " Rationale:\n"
-                           " Traditionally version 0 GASP tables were set"
-                           " so that font sizes below 8 ppem had no grid"
-                           " fitting but did have antialiasing. From 9-16"
-                           " ppem, just grid fitting. And fonts above"
-                           " 17ppem had both antialiasing and grid fitting"
-                           " toggled on. The use of accelerated graphics"
-                           " cards and higher resolution screens make this"
-                           " appraoch obsolete. Microsoft's DirectWrite"
-                           " pushed this even further with much improved"
-                           " rendering built into the OS and apps. In this"
-                           " scenario it makes sense to simply toggle all"
-                           " 4 flags ON for all font sizes.")
+              yield WARN, ("gaspRange {} value {} should be set"
+                           " to 1 (0x0F)").format(hex(key))
```

Comment:
Here we have 2 value-slots in the string but just one value is passed to `.format()`
Also the message is weird the way it is written here. Maybe what you would want to say here would be something like:

```python
"All flags in gaspRanges should be set to 1,"
" so the flags for {} should be 0x0F."
" Got {} instead.".format(hex(key), hex(value))
```

Reply:
Fixed